pubsub/tests/unit/pubsub_v1/test__gapic.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud.pubsub_v1 import _gapic
+
+
+class SourceClass(object):
+    def __init__(self):
+        self.x = "x"
+
+    def method(self):
+        return "source class instance method"
+
+    @staticmethod
+    def static_method():
+        return "source class static method"
+
+    @classmethod
+    def class_method(cls):
+        return "source class class method"
+
+    @classmethod
+    def blacklisted_method(cls):
+        return "source class blacklisted method"
+
+
+def test__gapic():
```

Comment:
Following the pattern in other test files, I would rename this test case to be explicit about what exactly is being tested with it (comes useful when reading test run output on failures). That nicely supplements the module name `test__gapic.py`, which tells where the unit under test is implemented.

```suggestion
def test_add_methods():
```